modules/core/client/directives/title.client.directives.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+angular.module('core').directive('title', ['$rootScope', '$timeout', '$interpolate', '$state',
+
+  function ($rootScope, $timeout, $interpolate, $state) {
+    return {
+      link: function (scope, element) {
+
+        var listener = function (event, toState) {
+
+          var title = (getWorkingState($state.$current));
+
+          $timeout(function () {
+            element.text(title);
+          }, 0, false);
+
+        };
+
+        function getWorkingState(currentState) {
```

Comment:
Probably should change the name of this function since it returns the `title` now.